azurerm/internal/services/subscription/subscription_resource_test.go

```diff
@@ -105,7 +114,13 @@ func (SubscriptionResource) Exists(ctx context.Context, client *clients.Client,
 		}
 		return nil, fmt.Errorf("retrieving Subscription Alias %q: %+v", id.Name, err)
 	}
-
+	subscriptionId := state.Attributes["subscription_id"]
+	// atags := state.Attributes["tags"]
+	tagsResp, err := client.Subscription.TagsClient.GetAtScope(ctx, "subscriptions/"+subscriptionId)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving tags from subscription %q: %+v", subscriptionId, err)
+	}
+	fmt.Println(tagsResp.Properties)
```

Comment:
This shouldn't be necessary. The exists and implicit read/import functionality will check the values for the tags.  The `Exists` function itself is simply to check the resource is present, not for any particular setting / configuration
```suggestion
```